### ISSUE TITLE
Work through Shoes manual - are the examples working?

### ISSUE THREAD

**Issue by noahgibbs** [scarpe-team/scarpe#461]
I'm pretty sure we do *not* support all the examples in the Shoes manual (docs/static/manual.md). Somebody should go through, add them to a directory under examples/legacy, and see how many of them work or don't. They'll probably want to start in not_checked rather than working since it's very likely they won't all work.

Eventually it would be really nice to write tests that make sure they do what they're supposed to do. But first, typing them in at all would be a solid start.

**Comment by BLSowmya** [comment]
is it okay to add few examples which are working and present in manual.md to  working directory?

**Comment by noahgibbs** [comment]
Absolutely. But it would be nice to have a specific directory of examples from the manual together, so that we'd know if they broke that the code in the manual isn't working :-) In the longer run, we'd really like people to be able to work through the manual successfully.
